doc/classes/Particles2D.xml

```diff
@@ -31,6 +31,7 @@
 		</member>
 		<member name="draw_order" type="int" setter="set_draw_order" getter="get_draw_order" enum="Particles2D.DrawOrder">
 			Particle draw order. Uses [code]DRAW_ORDER_*[/code] values. Default value: [code]DRAW_ORDER_INDEX[/code].
+			This controls the order in which individual particles are drawn. Index means particles are drawn according to their emission order [code](default)[/code]. [code]Lifetime[/code] means they are drawn in order of remaining lifetime.
```

Comment:
This line seems weird here. Why not add those details in the descriptions of the enum members themselves?

Reply:
It's good info to have in the tool tip popup though. It's really annoying to have to open the full class reference page to get the basic info you where hoping to find in the tool tip popup, and really annoying when the tool tip is just a useless repeat of the attribute name. Which is the main thing I'm trying to add more details to help with. Making tool tips more useful.

Reply:
But it's redundant -- that's what the documentation is for. You don't need to explain the options here as that just means you're now having to maintain two places whenever you want to change one description.